site/it/tutorials/quickstart/beginner.ipynb

```diff
@@ -91,7 +91,7 @@
         "colab_type": "text"
       },
       "source": [
-        "Note: La nostra comunità di Tensorflow ha tradotto questi documenti. Poichè questa traduzioni della comunità sono *best-effort*, non c'è garanzia che questa sia un riflesso preciso e aggiornato della [documentazione ufficiale in inglese](https://www.tensorflow.org/?hl=en). \n",
+        "Note: La nostra comunità di Tensorflow ha tradotto questi documenti. Poichè queste traduzioni sono *best-effort*, non è garantito che rispecchi in maniera precisa e aggiornata la [documentazione ufficiale in inglese](https://www.tensorflow.org/?hl=en). \n",
```

Comment:
Maybe "rispecchino" instead of "rispecchi", when I read it feels that this verb should be in the plural form

Reply:
If you change the disclaimer, can you also update the [it/README](https://github.com/tensorflow/docs/blob/master/site/it/README.md)?
Also, other files may need to be updated.

Reply:
Plural aligned into README and the following files already pushed: index.mb, advanced.ipynb, beginner.ipynb, classification.ipynb, and text_classification.ipynb.

I take this opportunity to ask a question: 
Does the translation disclaimer need to be present in each file?
Thank you. GB